notebooks/feature_engineering/labs/4_keras_adv_feat_eng.ipynb

```diff
@@ -233,9 +233,9 @@
    "outputs": [],
```

Comment:
<p>The training file from&nbsp;<code>introduction_to_tensorflow</code>&nbsp;has only 7332 rows whereas the one in the&nbsp;<code>feature_engineerin</code>&nbsp;folder has 53K+ rows. It is better to train with more data rather than repeat the small dataset of 7332 rows 30 times.</p><p> </p><p>So instead of doing this, please consider re-ordering the CSV_COLUMNS variable and keeping the volume of data the same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>